src/stringify.ts

```diff
@@ -64,7 +64,31 @@ const stringify = (
     tagString += `@${tag}${" ".repeat(tagTitleGapAdj || 0)}`;
   }
   if (type) {
-    tagString += gap + `{${type}}` + " ".repeat(tagTypeGapAdj);
+    const getUpdatedType = () => {
+      if (!isDefaultTag(tag)) {
+        return `{${type}}`
+      }
+
+      // The space is to improve readability in non-monospace fonts
+      if (type === '[]') return '[ ]'
+      if (type === '{}') return '{ }'
+
+      const squareBracketBoilerplateMatch = type.match(/^\[(.*)]$/)?.[1] || ''
+
+      const isAnObject = (value: string): boolean => /^{.*[A-z0-9_]+ ?:.*}$/.test(value)
+      const fixObjectCommas = (objWithBrokenCommas: string): string => objWithBrokenCommas.replace(/; ([A-z0-9_])/g, ', $1')
```

Comment:
This is needed since it is getting assigned to `type` instead of `name`.

Assigning to `type` seams to mean that objects get formatted as TS interfaces rather than as objects.